libs/libxerces-c/Makefile

```diff
@@ -10,7 +10,7 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=xerces-c
 PKG_MAIN_VER:=3.2
```

Comment:
Not sure what this is about?

Reply:
Looks like it's used later in the Install section. ¯\_(ツ)_/¯

Reply:
The install section looks strange especially since PKG_INSTALL is definied?
FreeBSD seems to fix it another way:
https://svnweb.freebsd.org/ports/head/textproc/xerces-c3/Makefile?revision=484139&view=markup#l44